### ISSUE TITLE
npm install on Dockerfile fails

### ISSUE THREAD

**Issue by Hazmi35** [akbaryahya/DockerGC#1]
I was building the Docker image with BuildKit, then this happened:

```
#6 10.32 Error: could not get uid/gid
#6 10.32 [ 'nobody', 0 ]
#6 10.32 
#6 10.32     at /usr/lib/node_modules/npm/node_modules/uid-number/uid-number.js:37:16
#6 10.32     at ChildProcess.exithandler (child_process.js:283:5)
#6 10.32     at emitTwo (events.js:126:13)
#6 10.32     at ChildProcess.emit (events.js:214:7)
#6 10.32     at maybeClose (internal/child_process.js:915:16)
#6 10.32     at Process.ChildProcess._handle.onexit (internal/child_process.js:209:5)
#6 10.32 TypeError: Cannot read property 'get' of undefined
#6 10.32     at errorHandler (/usr/lib/node_modules/npm/lib/utils/error-handler.js:205:18)
#6 10.32     at /usr/lib/node_modules/npm/bin/npm-cli.js:78:20
#6 10.32     at cb (/usr/lib/node_modules/npm/lib/npm.js:228:22)
#6 10.32     at /usr/lib/node_modules/npm/lib/npm.js:266:24
#6 10.32     at /usr/lib/node_modules/npm/lib/config/core.js:83:7
#6 10.32     at Array.forEach (<anonymous>)
#6 10.32     at /usr/lib/node_modules/npm/lib/config/core.js:82:13
#6 10.32     at f (/usr/lib/node_modules/npm/node_modules/once/once.js:25:25)
#6 10.32     at afterExtras (/usr/lib/node_modules/npm/lib/config/core.js:173:20)
#6 10.32     at Conf.<anonymous> (/usr/lib/node_modules/npm/lib/config/core.js:231:22)
#6 10.32 /usr/lib/node_modules/npm/lib/utils/error-handler.js:205
#6 10.32   if (npm.config.get('json')) {
#6 10.32                  ^
#6 10.32 
#6 10.32 TypeError: Cannot read property 'get' of undefined
#6 10.32     at process.errorHandler (/usr/lib/node_modules/npm/lib/utils/error-handler.js:205:18)
#6 10.32     at emitOne (events.js:116:13)
#6 10.32     at process.emit (events.js:211:7)
#6 10.32     at process._fatalException (bootstrap_node.js:391:26)
```

**Comment by Hazmi35** [comment]
It seems like npm doesn't like user `nobody` installing a package globally

**Comment by Hazmi35** [comment]
#2 Fixed this

**Comment by Hazmi35** [comment]
Huh, I accidently mentioned this issue in #3 lol, this is actually should be closed after #2 is merged.
